cdap-watchdog/src/main/java/co/cask/cdap/logging/save/KafkaLogWriterPlugin.java

```diff
@@ -158,7 +158,7 @@ public void init(Set<Integer> partitions) {
     countDownLatch = new CountDownLatch(1);
 
     if (partitions.contains(0)) {
-      LOG.info("Scheduling cleanup task");
+      LOG.debug("Scheduling cleanup task");
```

Comment:
This is probably not needed. @poornachandra  confirmed that this happens once, and is good to know.